src/base_classes/forte_options.cc

```diff
@@ -221,6 +233,90 @@ std::vector<double> ForteOptions::get_double_vec(const std::string& label) {
     return result;
 }
 
+void set_gen_list(const std::string& label, py::list val);
+void set_int_vec(const std::string& label);
+void set_double_vec(const std::string& label, const std::vector<double>& val);
+
+void ForteOptions::set_bool(const std::string& label, bool val) {
+    auto value_type = get(label);
+    if (value_type.second == "bool") {
+        set(label, py::bool_(val));
+    } else {
+        std::string msg = "Called ForteOptions::set_bool(" + label +
```

Comment:
I wonder if these error messages along with those in the get_… functions can be standardized into one function.

Reply:
Good point, done.